### ISSUE TITLE
[Feature]: IconButton missing label

### ISSUE THREAD

**Issue by AndreaKarlsen** [HelseVestIKT/hviktor-blazor#33]
### IconButton
Missing accessable name
<img width="1667" height="1217" alt="Image" src="https://github.com/user-attachments/assets/47232a6d-e294-4478-92cf-e1259a112170" />

Look to Designsystemet and add 
````html
<button aria-label="Icon only"></button>
````

**Comment by Cusatelli** [comment]
See comment in: https://github.com/HelseVestIKT/hviktor-blazor/issues/106#issuecomment-5476336879

**Comment by AndreaKarlsen** [comment]
@Cusatelli I do not consider this a lack of documentation and something we should not fix. We should not provide components that is not accessible and we should update the Blazor components to the latest designsystemet version.

**Comment by Cusatelli** [comment]
@AndreaKarlsen I agree, however, as I understood the issue it was simply missing an aria-label attribute? These are provided by the developer, and in this case it was not done so. The component supports it just fine.

Therefore this issue is not component related. But it's still a valid issue.
